tfjs-backend-webgpu/src/conv2d_mm_webgpu.ts

```diff
@@ -99,20 +98,49 @@ export class Conv2DMMProgram implements WebGPUProgram {
   }
 
   getUserCode(): string {
+    const coordASnippet = this.isChannelsLast ? `
+    let coord = vec4<i32>(
+      batch,
+      outRow * uniforms.stride[0] + uniforms.dilation[0] * WRow - uniforms.pad[0],
```

Comment:
How about Line 104,105 used as a string, so line 112, 113 can be simple?

Reply:
Done